category.html

```diff
@@ -0,0 +1,499 @@
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">TrendHub</h3>
+            <p class="product-description">Digital Marketing Hero Section</p>
+          </a>
+          <p class="item-price">$6.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">Thinky</h3>
+            <p class="product-description">Blog News Mobile App</p>
+          </a>
+          <p class="item-price">$18.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">ArtVerse</h3>
+            <p class="product-description">AI Tools Website Design</p>
+          </a>
+          <p class="item-price">$12.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">Artique</h3>
+            <p class="product-description">Creative Agency Website Design</p>
+          </a>
+          <p class="item-price">$12.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">MindTrix</h3>
+            <p class="product-description">AI Tools Website Design</p>
+          </a>
+          <p class="item-price">$5.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+
+        <!-- Product Item -->
+        <article class="product-item">
+          <a href="./product-details.html">
+            <img src="/" alt="Product Asset" />
+            <h3 class="product-title">Koki</h3>
+            <p class="product-description">Restaurant Admin Dashboard</p>
+          </a>
+          <p class="item-price">$20.00</p>
+          <button class="cart-btn">
+            <img src="/" alt="Shop icon" />
+          </button>
+        </article>
+      </section>
+
+      <!-- Pagination -->
+      <div class="pagination">
```

Comment:
I made it as div with links, because I googled and in every article it was suggested like that... 

https://www.w3schools.com/howto/howto_css_pagination.asp

I will leave it as it is for now and ask Liza for clarification.

Reply:
We've discussed this with Liza :) This solution that you've found is common but pretty old. For now, we have a better and more semantic solution such as a list with links. But, of course, you can ask for a deeper explanation at class

Reply:
Ah, I have understood now. I just need to wrap it as a list with links. 

#fixed